hbase-plugins/src/main/java/io/cdap/plugin/HBaseConfig.java

```diff
@@ -66,14 +72,26 @@ public HBaseConfig(String referenceName, String tableName, String rowField, @Nul
    * @throws IllegalArgumentException if the schema is null or not as valid JSON
    */
   public Schema getSchema() {
-    if (schema == null) {
-      throw new IllegalArgumentException("Schema cannot be null");
+    if (Strings.isNullOrEmpty(schema)) {
+      throw new IllegalArgumentException("Schema is either null or empty. It must be specified.");
```

Comment:
null or empty is technical. We can just say something like:

Schema must be provided.